payrix.gemspec

```diff
@@ -15,6 +15,5 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "faraday", "~> 0.14"
   spec.add_runtime_dependency "faraday_middleware", "~> 0.12"
-  spec.add_development_dependency "bundler", "~> 1.16"
```

Comment:
Let the `bundler` version be defined by the environment it is run in. Since this is a development dependency, it should have no impact to the functionality of the gem.